dir.targets

```diff
@@ -110,4 +110,15 @@
     <Message Importance="High" Condition="'$(ConfigurationErrorMsg)' != ''" Text="$(MSBuildProjectFullPath) ConfigurationErrorMessage: $(ConfigurationErrorMsg)" />
   </Target>
 
+  <Target Name="GenerateReferenceAssembly">
```

Comment:
Right now this will just be manual. @ericstj and I were discussing a few different things we would need to work through if we want to make it automatic in some way. And like you mentioned, the ref's aren't fully "standardized" right now, so it would cause a lot of churn at first.